contents/all.tsx

```diff
@@ -7,19 +7,40 @@ export const config: PlasmoContentScript = {
   matches: ["https://*/*"]
 }
 
+const DIV_ID = "plasmo-shadow-div"
 const SHADOW_STYLE_ID = "shadow-styling"
+
+// We use the getRootContainer to inject the shadow host in our own way
+export const getRootContainer = async () => {
+  const container = document.createElement("div")
+
+  container.id = "plasmo-shadow-container"
+
+  container.style.cssText = `
+    display: none;
+    z-index: 1;
+    position: absolute;
+  `
```

Comment:
I'll see if I can put the `display: none` also on the shadowHost element directly.

It'd be nice to have the global visibility indicator on the host outside of the shadow dom so that the shadow dom can be concerned exclusively with the styling when visible.

At least, that's what I'm thinking currently.